jax/nn/functions.py

```diff
@@ -22,6 +22,7 @@
 from jax import lax
 from jax import core
 from jax.scipy.special import expit
+from jax.scipy.special import logsumexp as _logsumexp
```

Comment:
Seems like we could just:
```
from jax.scipy.special import logsumexp
```
and save the extra alias below?

Reply:
I had started with that but the linter then complained that `logsumexp` was imported but not used.